README.md

```diff
@@ -11,7 +11,30 @@ Basical fcitx5 functions seems work after a simple setup.
 
 ## Install
 
-For packer.nvim:
+### rocks.nvim
+
+`~/.config/nvim/rocks.toml`:
+
+```toml
+[plugins]
+"fcitx5-ui.nvim" = "scm"
+```
+
+or:
+
+```vim
+:Rocks install fcitx5-ui.nvim
+```
+
+or:
+
+```sh
+$ luarocks --lua-version 5.1 --local --tree ~/.local/share/nvim/rocks install fcitx5-ui.nvim
+# ~/.local/share/nvim/rocks is the default rocks tree path
+# you can change it according to your vim.g.rocks_nvim.rocks_path
+```
```

Comment:
I would remove this section.
If you use luarocks to install, without adding an entry to the rocks.toml, rocks.nvim will remove it again on `:Rocks sync`.

Reply:
Right. Updated.